feature-libs/product-configurator/rulebased/cpq/cpq-configurator-normalizer.ts

```diff
@@ -303,7 +305,11 @@ export class CpqConfiguratorNormalizer
       }
 
       case Cpq.DisplayAs.INPUT: {
-        uiType = Configurator.UiType.STRING;
+        if (sourceAttribute?.dataType === Cpq.DataType.INPUT_STRING) {
+          uiType = Configurator.UiType.STRING;
+        } else {
+          uiType = Configurator.UiType.NOT_IMPLEMENTED;
+        }
```

Comment:
this does not seem to be related to 'move value from input field'. did you fix multiple issues with this PR?

Reply:
Yes. Several issues (directly or indirectly related to handling of input fields) are fixed with this pull request. All adjustments are mentioned in the pull request comment.